Rivet/Functions/Get-RivetConfig.ps1

```diff
@@ -87,48 +80,39 @@ function Get-RivetConfig
     {
         [CmdletBinding()]
         param(
-            [Parameter(Mandatory=$true)]
-            [string]
+            [Parameter(Mandatory)]
             # The name of the property to get.
-            $Name,
+            [String]$Name,
 
-            [Switch]
             # The configuration value is required.
-            $Required,
+            [switch]$Required,
 
             [Parameter(Mandatory,ParameterSetName='AsInt')]
-            [Switch]
             # Set the configuration value as an integer.
-            $AsInt,
+            [switch]$AsInt,
 
-            [Parameter(Mandatory,ParameterSetName='AsList')]
-            [Switch]
+            [Parameter(Mandatory,ParameterSetName='AsArray')]
             # Set the configuration value as a list of strings.
-            $AsList,
+            [switch]$AsArray,

```

Comment:
Usage changes?

Reply:
No. I thought `AsArray` was clearer what was being returned than `AsList`. I didn't want people to think a `Collections.Generic.List` was getting returned.